src/test/suite/fixtures/recorded/selectionTypes/clearPaint.yml

```diff
@@ -0,0 +1,28 @@
+languageId: plaintext
```

Comment:
I don't understand this test. Why is this not an error?

Reply:
Why would it? Are referring to my issue where all plain text tests fail?

Reply:
See above; the end of selection is not contained in a non-white space sequence, so should be an error